src/core/config.ts

```diff
@@ -85,7 +85,9 @@ export class Config {
       WARTHOG_GENERATED_FOLDER: path.join(this.PROJECT_ROOT, 'generated'),
       WARTHOG_RESOLVERS_PATH: [path.join(this.PROJECT_ROOT, 'src/**/*.resolver.ts')],
       WARTHOG_SUBSCRIPTIONS: 'false',
-      WARTHOG_VALIDATE_RESOLVERS: 'false'
+      WARTHOG_VALIDATE_RESOLVERS: 'false',
+      WARTHOG_KEEP_ALIVE_TIMEOUT_MS: 30000,
+      WARTHOG_HEADERS_TIMEOUT_MS: 5000
```

Comment:
I think since you changed this to not be an offset, we'd need to set this to a value larger than `WARTHOG_KEEP_ALIVE_TIMEOUT_MS`, right?

In looking at these pages:
- https://shuheikagawa.com/blog/2019/04/25/keep-alive-timeout/
- https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#connection-idle-timeout
 - https://nodejs.org/api/http.html#http_server_headerstimeout

It seems both of these values need to be > 60.  Should we do something like `61000` and `62000`?

Reply:
It's more about `headersTimeout > keepAliveTimeout`.  30s is a good keep alive (though if you want it to be 60s by default I can certainly make that change), but the key part of this is to ensure that the headers time out is greater than the keep alive.

Reply:
I pushed up the change to make headers timeout 60s, that was an oversight in my last commit.